mlflow/utils/search_utils.py

```diff
@@ -677,42 +677,6 @@ def _parse_list_from_sql_token(cls, token):
                 error_code=INVALID_PARAMETER_VALUE,
             )
 
-    @classmethod
-    def _get_comparison_for_model_registry(cls, comparison, valid_search_keys):
```

Comment:
The method is not used after tag based model / version search PR merged.

Reply:
Or you could say title the PR `Remove unused _get_comparison_for_model_registry`.